frontend/src/components/mine/ContactInfo/ViewMineManager.js

```diff
@@ -50,8 +50,21 @@ export class ViewMineManager extends Component {
 
   openModal(event, onSubmit, handleChange, handlePartySubmit, title) {
     event.preventDefault();
+
+    let mineManager = null;
+    let startDate = null;
+    let appointment = this.props.mine.mgr_appointment;
+    if (appointment != null && appointment.length > 0) {
+      mineManager = appointment[0].party_guid;
+      startDate = appointment[0].effective_date;
+    } 
```

Comment:
It would be preferable and more consistent with our existing code to use `const`. You could do it something like this:
```
const hasAppointment = appointment != null && appointment.length > 0;
const mineManager = hasAppointment ? appointment[0].party_guid : null;
const startDate = hasAppointment ? appointment[0].effective_date : null;
```

Reply:
Sounds good, I have made this change.